### ISSUE TITLE
Faces this error while using whatsapp. 

### ISSUE THREAD

**Issue by singhh9596** [florisboard/florisboard#1329]
#### Environment information
- FlorisBoard Version: 0.3.13 (56)
- Device: Xiaomi Redmi K20 Pro
- Android: 11 (cn=R sdk=30)

#### Features enabled
```kt
smartbar = true
suggestions = true
suggestions_clipboard = true
suggestions_next_word = true
glide = false
clipboard_internal = true
clipboard_history = false
```

#### Attached stacktrace files
<details>
<summary>1634551637718.stacktrace</summary>

```
java.lang.NullPointerException
	at dev.patrickgold.florisboard.ime.core.FlorisBoard$Companion.getInstance(FlorisBoard.kt:182)
	at dev.patrickgold.florisboard.ime.text.gestures.GlideTypingManager.setLayout(GlideTypingManager.kt:64)
	at dev.patrickgold.florisboard.ime.text.keyboard.TextKeyboardView.initGlideClassifier$lambda-48(TextKeyboardView.kt:1301)
	at dev.patrickgold.florisboard.ime.text.keyboard.TextKeyboardView.$r8$lambda$U7OJ4sHYwGnSHXM36JYIb8DgTag(Unknown Source:0)
	at dev.patrickgold.florisboard.ime.text.keyboard.TextKeyboardView$$ExternalSyntheticLambda1.run(Unknown Source:2)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7662)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:940)

```
</details>

**Comment by singhh9596** [comment]
Yes, I'm using island app to clone the florisBoard and whatsapp. So when I received the text on cloned whatsapp so I tried to reply the text from the status bar. Then only it got crashed. Btw this happened to me only once.

Another bug I face is, while sometimes typing the keyboard it won't respond to the touch and it needs to be force close (happens with non cloned version as well). I've turned of battery optimization but it still happens although its rare case scenario.
